.github/workflows/gradle-build-feature.yml

```diff
@@ -0,0 +1,19 @@
+name: Gradle Build Feature Branch
+on:
+  push:
+    branches: [develop]
```

Comment:
this will run twice, correct ?I mean gradle-build-development workflow will also be triggered on push to develop and this one as well ?

Reply:
Thank you for the review. It would run twice. My syntax was wrong. I have updated to be:
```
on:
  push:
    branches:
      - 'feature/**'
      - 'bug/**'
```